resources/files/ruby_219/rbconfig/rbconfig-powerpc-ibm-aix5.3.0.0.rb

```diff
@@ -0,0 +1,268 @@
+
```

Comment:
We're no longer building/shipping agents for AIX 5.3, so we may be able to drop this file.